com.adsbynimbus.nimbus/Runtime/Plugins/RTB.Tests/Properties/AssemblyInfo.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System.Reflection;
+﻿#if UNITY_EDITOR || UNITY_IOS || UNITY_ANDROID
```

Comment:
If the only code that needs to run is in the `#else` block you can simplify it by removing the else and changing the condition to `#if !UNITY_EDITOR && !UNITY_IOS && !UNITY_ANDROID`

Out of curiosity what was failing that is fixed by this compiler directive?